app/Http/Controllers/LoginController.php

```diff
@@ -32,9 +34,12 @@ public function postLogin(LoginRequest $request)
             return redirect()->route('home.index');
         }
         $input = $request->all();
-
         if (!empty($input)) {
             if (Auth::attempt(['email' => $input['email'], 'password' => $input['password']])) {
+                if (is_null(Auth::user()->email_verified_at)) {
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.ControlStructureSpacing.SpacingAfterOpen</br>**Severity**: error</br>**File**: app/Http/Controllers/LoginController.php</br>**Line**: 39</br>Blank line found at start of control structure